static/app/components/organizations/pageFilters/utils.tsx

```diff
@@ -71,14 +71,14 @@ export function getStateFromQuery(
  * Useful for extracting page filter properties from the current URL
  * when building another URL.
  */
-export function extractSelectionParameters(query) {
+export function extractSelectionParameters(query: Location['query']) {
```

Comment:
should we add `undefined` here?

Reply:
afaict we never use it when undefined right?

Reply:
oh wait, is it failing for that exact reason 😆

Reply:
I'll take a closer look

Reply:
yes that's why :)